assets/src/js/admin/onboarding-wizard/app/steps/your-cause/index.js

```diff
@@ -14,40 +14,60 @@ import ContinueButton from '../../../components/continue-button';
 import OtherIcon from '../../../components/icons/other';
+import SkipLink from '../../../components/skip-optin-link';
 
 // Import styles
 import './style.scss';
 
 const YourCause = () => {
-	const [ { configuration }, dispatch ] = useStoreValue();
+	const [{ configuration }, dispatch] = useStoreValue();
 
 	const userType = configuration.userType;
 	const causeType = configuration.causeType;
 
 	return (
 		<div className="give-obw-your-cause">
-			<h1>{ __( 'What does fundraising look like for you?', 'give' ) }</h1>
-			<CardInput values={ userType } onChange={ ( values ) => dispatch( setUserType( values ) ) } checkMultiple={ false } >
+			<h1>{__( '👋 Hi there! Tell us a little about your Organization.', 'give' )}</h1>
+			<p>{__( 'This information will be used to customize your experience to your fundraising needs.', 'give' )}</p>
+			<CardInput values={userType} onChange={( values ) => dispatch( setUserType( values ) )} checkMultiple={false}>
 				<Card value="individual">
 					<IndividualIcon />
-					<p>{ __( 'I\'m fundraising as an', 'give' ) }</p>
-					<strong>{ __( 'Individual', 'give' ) }</strong>
+					<p>{__( 'I\'m fundraising as an', 'give' )}</p>
+					<strong>{__( 'Individual', 'give' )}</strong>
 				</Card>
 				<Card value="organization">
 					<OrganizationIcon />
-					<p>{ __( 'I\'m fundraising within an', 'give' ) }</p>
-					<strong>{ __( 'Organization', 'give' ) }</strong>
+					<p>{__( 'I\'m fundraising within an', 'give' )}</p>
+					<strong>{__( 'Organization', 'give' )}</strong>
 				</Card>
 				<Card value="other">
 					<OtherIcon />
-					<p>{ __( 'My fundraising is', 'give' ) }</p>
-					<strong>{ __( 'Other', 'give' ) }</strong>
+					<p>{__( 'My fundraising is', 'give' )}</p>
+					<strong>{__( 'Other', 'give' )}</strong>
 				</Card>
 			</CardInput>
-			<h2>{ __( 'What is your cause?', 'give' ) }</h2>
-			<span className="screen-reader-text">{ __( 'What is your cause?', 'give' ) }</span>
-			<SelectInput testId="cause-select" value={ causeType } onChange={ ( value ) => dispatch( setCauseType( value ) ) } options={ getCauseTypes() } />
-			<ContinueButton testId="cause-continue-button" />
+
+			<div className="give-obw-optin-field">
+				<h2>{__( 'What are you fundraising for?', 'give' )}</h2>
+				<span className="screen-reader-text">{__( 'What type of cause is yours?', 'give' )}</span>
+				<SelectInput testId="cause-select" value={causeType} onChange={( value ) => dispatch( setCauseType( value ) )} options={getCauseTypes()} />
+			</div>
+
+			<div className="give-obw-community-field">
+				<h2>{__( '🌱 Would you like to join the GiveWP Community?', 'give' )}</h2>
+				<p>{__( 'If you opt-in, some data will be shared with our teams (so they can work their butts off to improve). We will also share some helpful info on fundraising, WordPress and our products from time to time.  And if you skip this, that’s okay! GiveWP will still work just fine.', 'give' )}</p>
```

Comment:
Change here to "By opting-in, you allow some basic data about how you use GiveWP to be used for us to improve the plugin for others. You also will receive emails from us with fundraising tips and more (which you can always unsubscribe from if you need to). If you skip this step, that's okay! GiveWP will still be setup for you no problem."

Reply:
Added in 27ea52c